builds/misc/packages-release.yaml

```diff
@@ -5,6 +5,13 @@
 # Variable 'WIN_PACKAGE_VERSION' was defined in the Variables tab
 name: $(VERSION)
```

Comment:
> Fix the bug when the release page uses a whole changelog instead of parsing correctly.

What do you mean by using a whole changelog?

Reply:
There was an issue where instead of parsing changelog.md from the ReleaseVersion-to-PreviousVersion for GitHub release page, the script will just dump the whole changelog.md content to the GitHub release page.

Reply:
Can you point out the part of the PR that fixed that?

Reply:
We fixed this by automating the tagging of the branch prior to the call.

Reply:
builds/misc/packages-release.yaml's `github_update_and_push`

Reply:
All that changed there is the checkout and the pull git commands. Can you elaborate?

Reply:
This is the part doing the tagging. How does this fix the issue mentioned in PR description:
> Fix the bug when the release page uses a whole changelog instead of parsing correctly.